polling_stations/apps/file_uploads/views.py

```diff
@@ -71,10 +89,24 @@ def get(self, request, *args, **kwargs):
 
     def get_context_data(self, **context):
         context["council"] = (
-            Council.objects.all()
+            Council.objects.using(LOGGER_DB_NAME)
+            .all()
             .exclude(council_id__startswith="N09")
             .get(council_id=self.kwargs["gss"])
         )
+        upcoming_election_dates = EveryElectionWrapper(
+            council_id=self.kwargs["gss"]
+        ).get_future_election_dates()
+
+        # If the list returns no items, flag that there are no upcoming elections
+        # that we know about.
+        context["NO_UPCOMING_ELECTIONS"] = bool(upcoming_election_dates)
```

Comment:
Would this make more sense as `context["HAS_UPCOMING_ELECTIONS"] = bool(upcoming_election_dates)`. At the moment if `upcoming_election_dates` is an empty list then `context["NO_UPCOMING_ELECTIONS"]` will be `False`, which I find unintuitive. But might just be me :)

Reply:
Yep, I think you're right. Maybe this is why I made the mistake about showing the button when no upcoming?